### ISSUE TITLE
[OCD ISSUE] Standardize Config

### ISSUE THREAD

**Issue by sxb1n9** [RainbowMiner/RainbowMiner#1881]
Standardize config files (ALL) to use "#" and "text" and stop mixing on different variables. 

Mostly and OCD thing but I am sometimes confused on what to put where. 

Thanks,

**Comment by RainbowMiner** [comment]
Yes, you are totally right. That's a relict that annoys me all the time, too. I'll add double quotes to all variables.
